tools/analyzer_plugin/lib/src/diagnostic/missing_cascade_parens.dart

```diff
@@ -1,6 +1,6 @@
 // ignore: deprecated_member_use
 import 'package:analyzer/analyzer.dart'
-    show CompileTimeErrorCode, NodeLocator, StaticTypeWarningCode, StaticWarningCode;
```

Comment:
The codes below were moved to `CompileTimeErrorCode`: https://github.com/dart-lang/sdk/commit/8de8a1c226bf227c485f0e024e225fbd248139c5#diff-ff9cb85838db7a802ac5aa9c55f26d86

Reply:
These don't exist in the lowest allowed version of the `analyzer` package; that dependency needs to be raised in order to use these.

Reply:
Ohh yep, thanks for catching that!